aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -58,94 +55,23 @@ spec:
 
   - name: ClusterName
     type: Input
-    description: What is the name of the cluster?
+    description: What do you want to name the cluster?
 
-  - name: NodeAutoScalingGroupDesiredSize
```

Comment:
why are NodeAutoScalingGroupDesiredSize and NodeInstanceType removed. IMO they are needed even if the cluster is basic else this just becomes another hello world kind of thing. Else user would have to change those manually

Reply:
I've added the scaling code back